src/content.ts

```diff
@@ -24,15 +24,48 @@ import {
     addContentStateChangedListener,
 } from "@src/content/state_content"
 
+import { CmdlineCmds } from "@src/content/commandline_cmds"
+import { EditorCmds } from "@src/content/editor"
+
+import { getAllDocumentFrames } from "@src/lib/dom"
+
+import state from "@src/state"
+import { EditorCmds as editor } from "@src/content/editor"
+/* tslint:disable:import-spacing */
+
+config.getAsync("disable").then(async TRI_DISABLE => {
 // Set up our controller to execute content-mode excmds. All code
 // running from this entry point, which is to say, everything in the
 // content script, will use the excmds that we give to the module
 // here.
-import * as controller from "@src/lib/controller"
-import * as excmds_content from "@src/.excmds_content.generated"
-import { CmdlineCmds } from "@src/content/commandline_cmds"
-import { EditorCmds } from "@src/content/editor"
-import * as hinting_content from "@src/content/hinting"
+const controller = await import("@src/lib/controller")
+const excmds_content = await import("@src/.excmds_content.generated")
+const hinting_content = await import("@src/content/hinting")
+// Hook the keyboard up to the controller
+const ContentController = await import("@src/content/controller_content")
+// Add various useful modules to the window for debugging
+const commandline_content = await import("@src/content/commandline_content")
+const convert = await import("@src/lib/convert")
+const dom = await import("@src/lib/dom")
+const excmds = await import("@src/.excmds_content.generated")
+const finding_content = await import("@src/content/finding")
+const itertools = await import("@src/lib/itertools")
+const messaging = await import("@src/lib/messaging")
+const State = await import("@src/state")
+const webext = await import("@src/lib/webext")
+const perf = await import("@src/perf")
+const keyseq = await import("@src/lib/keyseq")
+const native = await import("@src/lib/native")
+const styling = await import("@src/content/styling")
```

Comment:
I expected this not to have side-effects if it wasn't called, but that doesn't seem to be the case. (the root element of the page gets a TridactylThemeDefault class)

Reply:
9a99c21b one of the sillier things I've done for a while